spec/system/creating_a_journal_spec.rb

```diff
@@ -70,6 +69,19 @@
         click_button "Create"
         expect(page).to have_content "Pending Journal"
       end
+
+      context "with an order detail more than 90 days old", :js do
+        before do
+          reviewed_order_detail.fulfilled_at = 95.days.ago
+          reviewed_order_detail.save
+          visit new_facility_journal_path(facility)
+        end
+
+        it "has a 90 day pop up" do
+          click_button "Create"
+          expect(page).to have_content "90-Day Justification"
```

Comment:
What happens after clicking "Ok" here?

Reply:
@giladshanan just realized I forgot this one. Gotta head out for today.